hazelcast-jet-all/src/test/java/com/hazelcast/jet/server/JetCommandLineTest.java

```diff
@@ -112,6 +116,16 @@ public void before() {
         sourceMap = jet.getMap(SOURCE_NAME);
         IntStream.range(0, ITEM_COUNT).forEach(i -> sourceMap.put(i, i));
         sinkList = jet.getList(SINK_NAME);
+        assertTrueEventually(() -> {
+            if (!isJarFileExists()) {
+                createJarFile();
+            }
+            assertTrue(isJarFileExists());
+        });
+    }
+
+    public boolean isJarFileExists() {
+        return testJobJarFile.toFile().exists();
```

Comment:
Very minor comment - is not more clear to use `Files.exists(testJobJarFile);`?

Reply:
While we're at it, `Files.isFile()` makes more sense.